src/ashwat.cpp

```diff
@@ -20,6 +20,7 @@
 #include "cnglob.h"
 
 #include "ashwat.h"
+#include <climits>
```

Comment:
I didn't spot any place where this is being used? 
By the way, you're "adding climits header," not "adding climits library." :)